setup.py

```diff
@@ -13,24 +15,62 @@
 with open(os.path.join(BASE_DIR, "deflate.c"), "r") as src:
     version = re.match(r'.*#define MODULE_VERSION "(.*?)"', src.read(), re.S).group(1)
 
-cpu = "arm" if platform.machine().startswith("arm") else "x86"
+libdeflate_found = False
+
+try:
+    from pkgconfig import installed as pkgconfig_installed
+    from pkgconfig import parse as pkgconfig_parse
+except ImportError:
+    # pkgconfig is not installed. It will be installed by setup_requires.
+    pass
+else:
+    def pkgconfig_installed_check(lib, required_version, default):
```

Comment:
what would be the point of `default=True`?

Reply:
I left it there by accident. Removed